src/Spice86.Core/Emulator/InterruptHandlers/Input/Mouse/IMouseDevice.cs

```diff
@@ -47,11 +47,6 @@ public interface IMouseDevice : IIOPortHandler {
     /// </summary>
     MouseEventMask LastTrigger { get; }
 
-    /// <summary>
```

Comment:
It's important to keep this - as the sample rate is supposed to be configurable.

`Setting the Interrupt Rate for the InPort
Mouse
If you are using an InPort Mouse, you can use one of the
following command line switch settings to specify the
in terrupt rate for the mouse:
Switch Setting
/RO
/Rl
/R2
/R3
/R4
Interrupt Rate
disabled
30Hz (default)
50Hz
100Hz
200Hz`

[MS-DOS Mouse driver reference](https://archive.org/details/bitsavers_microsoftm1186MicrosoftMouseProgrammersReferenceGu_6193519)

Reply:
@maximilien-noal I removed it because it was neither set nor read. Should it still be kept?

Reply:
I added it back and force pushed the changes